### DISCUSSION TITLE
create the nzp for all consoles :)

### DISCUSSION THREAD

**Discussion by Ibrahimprox1** [nzp-team/nzportable#1031] [Other Content Releases]
Hello, I want to be a member of the nzp developers, and the idea I have is that nzp is on platforms such as: ps2, ps3, ps4 and ps5 and xbox 360 and xbox one and series, because of this I can help you with that The issue of optimization and bugs, I am a beginner or novice but I know how to optimize games, so I can make a program for PS2 or PS2 emulators that is well optimized and the Xbox 360 one too, if you want mobile phones and tables I can do it and help you, plus I know how to do those things and I will adapt Sincerely, Ibrahim :)

Note: and I can do it on more available devices and optimize it very well and put optimization technologies on it

**Comment by Riuk1z** [comment]
A Steam version with a workshop and the Spanish language would also be good because there are also fans of the game but they only speak Spanish.

**Comment by Riuk1z** [comment]
I hope you make it my friend one day
